td_load/google_sheet/config/daily_load.yml

```diff
@@ -16,12 +15,20 @@ in:
     datetime_render_option: FORMATTED_STRING
     key: spreadsheet_key_here
     # https://docs.treasuredata.com/display/public/INT/Google+Sheets+Export+Integration#Find-the-Spreadsheet-Key
-    
+
     null_string: '\N'
     default_typecast: strict
     columns:
-        - {name: test_name, type: string}
-        - {name: test_index, type: string}
+      - name: id
+        type: long
```

Comment:
I faced the error when I used `number`. So that add `columns` sample for several types.